framework-support/jobrunr-quarkus-extension/quarkus-jobrunr/src/main/java/org/jobrunr/quarkus/autoconfigure/storage/JobRunrInMemoryStorageProviderProducer.java

```diff
@@ -13,7 +13,7 @@ public class JobRunrInMemoryStorageProviderProducer {
     @Produces
     @DefaultBean
     @Singleton
-    //TODO: make it return correct storageprovider based on Extensions (mongo, redis, elasticsearch)?
+    //TODO: make it return correct storageprovider based on Extensions (mongo)?
```

Comment:
Isn't this already done in `org.jobrunr.quarkus.extension.deployment.JobRunrExtensionProcessor#storageProvider`?